WordPress/Classes/Plugins/Views/PluginDetailsView.swift

```diff
@@ -53,14 +53,14 @@ struct PluginDetailsView: View {
         // TODO: Use `shortDescription`
         self.pluginInfo = .init(name: plugin.name, author: plugin.author, shortDescription: plugin.author)
         self.service = service
-        _viewModel = StateObject(wrappedValue: .init(slug: slug, service: service))
+        _viewModel = StateObject(wrappedValue: .init(service: service))
```

Comment:
Hey, just to confirm, the way it works now is that the ViewModel is stateless and can be re-used for different plugins, correct?

I typically use [id](https://developer.apple.com/documentation/swiftui/view/id(_:)) to ensure the view is re-create when a new object is displayed, but that is also a common approach.